client/asset/btc/btc.go

```diff
@@ -1779,7 +1837,7 @@ func (btc *baseWallet) PreSwap(req *asset.PreSwapForm) (*asset.PreSwap, error) {
 	}
 
 	return &asset.PreSwap{
-		Estimate: est,
+		Estimate: est, // may be nil so we can present options, which in turn affect estimate feasibility
```

Comment:
Are other consumers aware of this change? (FE, etc)

Reply:
When making the same change for DCR recently, the `setFeeEstimates` function for the markets page was updated: https://github.com/decred/dcrdex/pull/2103/files#diff-1e4f8de7bbcd6448186a65982ea5e8fe3d7b64b982ed09520ff884829db5221aR1806
I didn't come across any other places to change at the time.